Assets/MRTK/Core/Utilities/WebRequestRest/Rest.cs

```diff
@@ -100,11 +95,11 @@ public static async Task<Response> PostAsync(string query, Dictionary<string, st
         /// <param name="timeout">Optional time in seconds before request expires.</param>
         /// <param name="readResponseData">Optional bool. If its true, response data will be read from web request download handler.</param>
         /// <returns>The response data.</returns>
-        public static async Task<Response> PostAsync(string query, WWWForm formData, Dictionary<string, string> headers = null, int timeout = -1, bool readResponseData = false)
+        public static async Task<Response> PostAsync(string query, WWWForm formData, Dictionary<string, string> headers = null, int timeout = -1, CertificateHandler certificateHandler = null, bool readResponseData = false)
```

Comment:
For this function and all of the other ones that take CertificateHandler as an optional thing, make it so that CertificateHandler is the last parameter i.e. instead of this:

```c#
 public static async Task<Response> PostAsync(string query, WWWForm formData, Dictionary<string, string> headers = null, int timeout = -1, CertificateHandler certificateHandler = null, bool readResponseData = false)
```

to this:

```c#
 public static async Task<Response> PostAsync(string query, WWWForm formData, Dictionary<string, string> headers = null, int timeout = -1, bool readResponseData = false, CertificateHandler certificateHandler = null)
```

The reason is that the first way breaks existing callers (i.e imagine someone who calls this way:

```c#
PostAsync(query, formData, headers, 100 /*ms*/, true /*readReponseData*/) 
```

This person is now broken because the bool that used to work is swapped for a different thing.

Reply:
I can add this to out coding guidelines to so that we can just point to a link going forward.

Reply:
For cases where you access it using named params it doesn't affect you but if you do the 'classic' way it can bite

Reply:
Done & updated XML documentation